CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Contributing to Sharezone
+_Note: The `CONTRIBUTING.md` is in the process of making. See [#28](https://github.com/SharezoneApp/sharezone-app/issues/28) for the missing sections._
+
+## Welcome
+We love, that you are interested in contributing to Sharezone 💙 There are many ways to contribute and we appreciate all of them. This document gives a rough overview of how you can contribute to Sharezone and which steps you need to follow to set up the development environment.
+
+* [How to set up your development environment](#how-to-set-up-your-development-environment)
+
+If you have ans questions, please join our [Discord](https://sharezone.net/discord).
+
+## How to set up your development environment
+If you would like to contribute 
+
+### Operating system
+_tl:dr: We recommend macOS._
+
+You can use the operating system you like. But we recommend to use macOS because you might have some issues with other operating systems.
+
+**Known issues:**
```

Comment:
@Jonas-Sander Are we supporting (supporting = we know it's working for the platform and willing to fix issues) all platforms for the Sharezone CLI (like macOS, ChromeOS, etc.)? Or do we only support macOS and (maybe) Windows? If we only supported specific operating systems, we should add this to our known issues, like:

> Sharezone CLI is only supported for macOS. Although, the CLI should work for other operating system but might have some issues. See here (a link to all GitHub tickets for specifc label for the CLI) for all known issues with the CLI for other operating systems.

Reply:
The CLI should support Windows and macOS and I think it does right now.

Reply:
I added the following point to `Known issues`:

> Sharezone CLI has official support for macOS and Windows (should also work with other operating systems, but might cause problems in some cases)